docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing Guidelines
+
+*We're so excited that you're interested in contributing to Neurobagel!* :partying_face:
+
+We appreciate all contributions, and hope that the below guidelines will make it as easy as possible for you to contribute to the [Neurobagel codebase](https://github.com/neurobagel) and to ensure your contribution can be easily integrated.
+
+## Identifying an issue to work on
+
+The easiest way to get started contributing is to explore the list of open issues in one of our [GitHub repositories](https://github.com/orgs/neurobagel/repositories). 
+For example, see the [open issues for the Neurobagel API](https://github.com/neurobagel/api/issues).
+
+Regardless of if you would like to contribute a feature, bug fix, or another type of change, we recommend that you first either
+
+- **If an issue exists**: 
+Comment on the open issue to indicate you would like to work on it, along with any clarification/implementation questions you have
+    - Check that no one is currently [assigned](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users) to the issue, 
+    which indicates it is already being worked on
+- **If an issue does not exist**: 
+Open a new issue describing your proposed change and why it is necessary/beneficial. 
+The more detail here, the better!
+
+This allows members of the Neurobagel developer team to confirm that you will not be overlapping with currently active work and that everyone is on the same page about the task to be accomplished.
+
+If you would like to contribute but are not sure where to start, we recommend looking for open issues with the following labels:
```

Comment:
should we mention GH discussions as well? Do we have those enabled anywhere? Maybe one central discussion for very generic questions?

Reply:
OK, I agree!